app/views/quota.html

```diff
@@ -84,11 +84,11 @@ <h3 class="text-center">Memory <small>Limit</small></h3>
                         <tr ng-repeat="(resourceType, specMax) in quota.spec.quota.hard"
                             ng-if="resourceType !== 'resourcequotas'"
                             ng-class="{
-                              warning: (quota.status.total.used[resourceType] | usageValue) >= (quota.status.total.hard[resourceType] | usageValue)
+                              warning: isAtLimit(quota, resourceType)
```

Comment:
i wonder if there is another visual way to treat rows where the max is set to zero, gray/disabled?  it feels like something special to me because you are being completely denied the use of this resource